meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -467,6 +467,7 @@ WKS_FILE_DEPENDS:append:stm32mp1common = " e2fsprogs-native"
 MACHINE_FEATURES:remove:stm32mp1common = "fw-update"
 PREFERRED_PROVIDER_u-boot-default-script:stm32mp1common = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script:sota:stm32mp1common = "u-boot-ostree-scr-fit"
+SOTA_CLIENT_FEATURES:append:stm32mp1common = " ubootenv"
```

Comment:
For this one I think it is better to have it set per machine specific override, as we might have -sec boards (containing stm32mp1common) which won't use u-boot env (but fiovb instead).